test/extensions/geoip_providers/maxmind/geoip_provider_test.cc

```diff
@@ -265,6 +333,53 @@ TEST_F(GeoipProviderTest, ValidConfigCityMultipleLookups) {
   expectStats("city_db", 2, 2);
 }
 
+TEST_F(GeoipProviderTest, DbReloadedOnMmdbFileUpdate) {
+  const std::string config_yaml = R"EOF(
```

Comment:
might be able to fix with `constexpr absl::string_view` , example in https://github.com/envoyproxy/envoy/commit/97691568ae72281632ad8fff0675f931a80dca22

Reply:
same issue in other `fmt::format` use cases

Reply:
luckily is only one usage :)